packages/fetchai/contracts/oracle/contract.py

```diff
@@ -22,13 +22,13 @@
 import logging
 from typing import Any, Dict
 
+from ethereum_crypto import EthereumApi
```

Comment:
We need to test how this works with mypy in a standalone project...

Reply:
removed